stories/DayPicker.js

```diff
@@ -52,12 +52,11 @@ storiesOf('DayPicker', module)
   .addWithInfo('with custom day size', () => (
     <DayPicker daySize={50} />
   ))
-  .addWithInfo('more than one month', () => (
-    <DayPicker numberOfMonths={2} />
+  .addWithInfo('single month', () => (
+    <DayPicker numberOfMonths={1} />
   ))
   .addWithInfo('vertical', () => (
     <DayPicker
-      numberOfMonths={2}
```

Comment:
Similarly, having these explicit ensures that a change of the default won't suddenly make these stories useless.